pkg/scheduler/apis/config/types.go

```diff
@@ -291,6 +291,39 @@ func (p *Plugins) Apply(customPlugins *Plugins) {
 	p.Unreserve = mergePluginSets(p.Unreserve, customPlugins.Unreserve)
 }
 
+// IsSuperset returns true if and only if plugins is a superset of another.
+func (p *Plugins) IsSuperset(plugins *Plugins) bool {
```

Comment:
```Contains``` or ```Has``` is more common

Reply:
Done

Just follow the set API here https://github.com/kubernetes/kubernetes/pull/87134/files#diff-b483008d22340548275bf45d61561d1dR324